Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.iOS/Logics/PinLogic.cs

```diff
@@ -106,7 +107,7 @@ void OnInfoTapped(object sender, GMSMarkerEventEventArgs e)
             // lookup pin
             var targetPin = LookupPin(e.Marker);
 
-            // only consider event handled if a handler is present. 
+            // only consider event handled if a handler is present.
```

Comment:
Please don't contain "not the purpose" changes.
Code cleaning is "not purpose" for this PR.